README.md

```diff
@@ -1,4 +1,64 @@
-# block-explorer
+# Generic Block Explorer
+This is backend and API for block-explorer application  
+It was created to for easily visualise and inspect data from https://github.com/insolar/insolar  
+The main idea is to get records stored in insolar using it's API and save it in relational-style format  
+It doesn't know anything about smart contracts  
+
+## Requirements
+ - go 1.14
+ - PostgeSQL 11+
+
+## How to start
+#### Resolve dependencies and build
+```
+make all
+```
+#### Create tables in DB
+```
+make migrate
```

Comment:
make migrated needs a config file. so config file is needed and needs to be mentioned